undebug/main.py

```diff
@@ -35,89 +37,85 @@ def bgp():
     @bgp.command('allow-martians')
     def allow_martians():
         """BGP allow martian next hops"""
-        command = 'sudo vtysh -c "no debug bgp allow-martians"'
+        command = ["sudo", "vtysh", "-c", "no debug bgp allow-martians"]
         run_command(command)
 
     @bgp.command()
     @click.argument('additional', type=click.Choice(['segment']), required=False)
     def as4(additional):
         """BGP AS4 actions"""
-        command = 'sudo vtysh -c "no debug bgp as4'
+        command = ["sudo", "vtysh", "-c", "no debug bgp as4"]
         if additional is not None:
-            command += " segment"
-        command += '"'
+            command[-1] += " segment"
         run_command(command)
 
     @bgp.command()
     @click.argument('prefix', required=True)
     def bestpath(prefix):
         """BGP bestpath"""
-        command = 'sudo vtysh -c "no debug bgp bestpath %s"' % prefix
+        command = ["sudo", "vtysh", "-c", "no debug bgp bestpath %s" % prefix]
```

Comment:
>prefix [](http://example.com/codeflow?start=71&length=6)

Need to validate prefix. Suggest checking it only contains numbers, alphabets, `. : /`

Reply:
This comment is still applicable.